scripts/prepare_regions_data/regions_data.ts

```diff
@@ -15,18 +15,21 @@ import {
 
 const { state_dataset, state_county_map_dataset } = US_STATE_DATASET;
 
+// used in the script to validate the generators
+export const stateFipsToUrlSegment: { [index: string]: string } = {};
+export const countyFipsToUrlSegment: { [index: string]: string } = {};
+export const metroAreaFipsToUrlSegment: { [index: string]: string } = {};
+
 function buildStates(): State[] {
-  return map(
-    state_dataset,
-    stateInfo =>
-      new State(
-        stateInfo.state,
-        stateInfo.state_url_name,
-        stateInfo.state_fips_code,
-        stateInfo.population,
-        stateInfo.state_code,
-      ),
-  );
+  return map(state_dataset, stateInfo => {
+    stateFipsToUrlSegment[stateInfo.state_fips_code] = stateInfo.state_url_name;
+    return new State(
+      stateInfo.state,
+      stateInfo.state_fips_code,
+      stateInfo.population,
+      stateInfo.state_code,
+    );
```

Comment:
FWIW- If you wanted you could drop the separate validation code and the `stateFipsToUrlSegment` maps and just do the validation here:

```
const s = new State(...)
assert(s.urlSegment === stateInfo.state_url_name, "Computed URL segment must match persisted one.");
return s;
```

and same for counties / metros...  I don't feel strongly though.

Reply:
ah, good idea.  but, yeah, maybe I'll save that if I have a reason to touch it later.